instances/devhub.near/widget/devhub/entity/proposal/AcceptedTerms.jsx

```diff
@@ -6,47 +6,43 @@ State.init({
   proposalBlockHeight: null,
 });
 
-const proposalId = props.proposalId;
+const instance = props.instance ?? "";
 
-const QUERYAPI_ENDPOINT = `https://near-queryapi.api.pagoda.co/v1/graphql`;
-const fetchGraphQL = (operationsDoc, operationName, variables) => {
-  return asyncFetch(QUERYAPI_ENDPOINT, {
-    method: "POST",
-    headers: { "x-hasura-role": "${REPL_INDEXER_HASURA_ROLE}" },
-    body: JSON.stringify({
-      query: operationsDoc,
-      variables: variables,
-      operationName: operationName,
-    }),
-  });
-};
+const { cacheUrl } = VM.require(`${instance}/widget/config.data`);
```

Comment:
what happens here if instance is ""?

Reply:
If we forget to pass the instance it would break the UI and the tests.